core/src/components/card/card.ionic.scss

```diff
@@ -4,15 +4,15 @@
 // --------------------------------------------------
 
 :host {
-  --background: #{globals.$ionic-color-neutral-0};
-  --border-radius: #{globals.$ionic-border-radius-rounded-small};
+  --background: #{globals.$ionic-color-base-white};
+  --border-radius: #{globals.$ionic-border-radius-100};
 
-  @include globals.padding(globals.$ionic-space-base);
+  @include globals.padding(globals.$ionic-space-400);
   @include globals.border-radius(var(--border-radius));
 
   display: block;
 
-  border: #{globals.$ionic-border-size-small} solid #{globals.$ionic-color-neutral-50};
+  border: #{globals.$ionic-border-size-025} solid #{globals.$ionic-color-neutral-500};
```

Comment:
According to UX/UI team almost all color values changed (this had already warned about this, the first color draft was not final). These new tokens should be the final ones for a long time now.

For now, I just kept using the same equivalent tokens (10 is now 100, etc), that’s why we have differences on the components we did already, as all the colors changed a bit.

Regarding Figma component files showing different values on the tokens, I already contacted Beth Thomas about this, its probably because she didn’t yet propagated the changes on their side.

But the Design Tokens should always represent the source of truth!